components/org.wso2.carbon.event.outbound.adapter.websubhub/src/main/java/org/wso2/carbon/event/outbound/adapter/websubhub/util/WebSubHubEventAdapterConstants.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Copyright (c) 2022, WSO2 LLC. (http://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.event.outbound.adapter.websubhub.util;
+
+/**
+ * Keep constants required by the WebSubHub Event Adapter.
+ */
+public class WebSubHubEventAdapterConstants {
+
+    public static final String ADAPTER_MESSAGE_TENANT_DOMAIN = "tenant-domain";
+    public static final String EVENT_ISSUER = "Asgardeo";
```

Comment:
Since here we are planning to use this service as a common adaptor for both asgardeo and on-prem product, don't we need to use a configurable issuer?